dbt_project.yml

```diff
@@ -2,22 +2,23 @@ name: 'medicare_cclf_connector'
 version: '1.0.0'
 config-version: 2
 
-# This setting configures which "profile" dbt uses for this project.
-profile: 'default'                       # configuration required
+profile: 'default'
 
-vars:
-  input_database: medicare_claims       # configuration required
-                                        # name of the database where sources feeding this project are stored
-
-  input_schema:  cclf                   # configuration required
-                                        # name of the schema where sources feeding this project is stored
 
-  output_database: tuva                 # configuration required
-                                        # name of the database where output of this project should be written
+## Variables affecting the configuration of this project and the tuva packages are below.
+## Many config variables have default values and don't need to be set.  The recommended settings to configure are
+##  uncommented below.  Other possible configurations are listed but commented.
+vars:
+  source_name: cclf                           # Name identifying the source of this cclf data.  Will be written to data_source columns in output tables.
 
-  output_schema: claims_input           # configuration required
-                                        # name of the schema where output of this project should be written
+#### Input Configurations
```

Comment:
The "#" are throwing me off. In markdown, one "#" indicates Header 1, "##" Header 2, "###" Header 3, etc. We should be consistent even if we don't follow the markdown syntax.

Reply:
yeah agreed its a little weird, we're planning a rework of the readme and dbt_project.yml driven by aaron as a separate pr after this is merged.